client/src/pages/Home.js

```diff
@@ -21,7 +44,7 @@ const Home = (props) => {
         <div>
              <Box sx={{ '& > button': { m: 1 } }} className="searchbar-container">
                 <CustomSearchBar />
-                <IconButton aria-label="add" size="large" style={{marginTop: '18px'}} onClick={changeToReviewScreen}>
+                <IconButton aria-label="add" style={{marginTop: '18px'}} onClick={changeToReviewScreen}>
```

Comment:
Can you add a TODO on top to change this from a page to a modal in the future ?

Reply:
can u get on a call and discuss these thx